components/Article/Codebox/__tests__/index.test.tsx

```diff
@@ -37,12 +37,37 @@ describe('Codebox component (one lang)', (): void => {
       'writeText'
     );
 
-    const buttonElement = screen.getByText('components.codeBox.copy');
+    const buttonElement = await screen.findByTestId('copy');
     await user.click(buttonElement);
 
     expect(navigatorClipboardWriteTextSpy).toHaveBeenCalledTimes(1);
     expect(navigatorClipboardWriteTextSpy).toHaveBeenCalledWith(code);
   });
+
+  it('should copy content with textToCopy', async () => {
+    const user = userEvent.setup();
+
+    const textToCopy = ['Example code'];
+
+    render(
+      <IntlProvider locale="en" onError={() => {}}>
+        <Codebox textToCopy={textToCopy}>
+          <pre className="language-js">{code}</pre>
+        </Codebox>
+      </IntlProvider>
+    );
+
+    const navigatorClipboardWriteTextSpy = jest.spyOn(
+      navigator.clipboard,
+      'writeText'
+    );
+
+    const buttonElement = await screen.findByTestId('copy');
```

Comment:
I thought the usage of `screen` was discouraged.

Reply:
this tripped up our team for a while too. but the maintainer actually says screen is peferred from a code organization standpoint. less keeping destructures up to date per test

https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen